.babelrc

```diff
@@ -1,4 +1,18 @@
 {
-  "presets": ["env", "react"],
-  "plugins": ["transform-object-rest-spread"]
+  "presets": [
+    "@babel/preset-react",
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry",
+        "targets": {
+          "ie": 11
+        }
+      }
+    ]
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-object-rest-spread",
+    "transform-class-properties"
+  ]
```

Comment:
Does this work if we rename it to `.babelrc.json` and if so, can we do that and then `require` it when we use the same config elsewhere?

Reply:
seems to 👍